Scripts/pre-commit.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -eu
```

Comment:
Yes, it helps a lot when I only run one spec and I don't want to commit that to the server.